blog post to announce 2.1.0 (#219)
MDAnalysis/MDAnalysis.github.io: _posts/2022-03-06-release-2.1.0.md

# Load a pure water system
universe = mda.Universe(PSF_TRICLINIC, DCD_TRICLINIC)

diel = DielectricConstant(universe.atoms)
diel.run()
print(diel.results)
```

```
{'M': array([ 1.54082059, -2.03885814,  3.37261771]), 'M2': array([ 5.71970697, 13.45815613, 18.24527209]), 'fluct': array([3.34557887, 9.3012136 , 6.87072189]), 'eps': array([2.47772633, 5.10830197, 4.0347653 ]), 'eps_mean': 3.873597866567474}
```

### R/S chirality is read from RDKit objects

[Richard Gowers][] made the reading of RDKit molecules aware of the R/S chirality when the information is available. The information can be accessed with the `AtomGroup.chirality` attribute and be used for selections.

```python
import MDAnalysis as mda
from rdkit import Chem

molecule = Chem.MolFromSmiles('CC[C@H](C)[C@H](C(=O)O)N')
universe = mda.Universe(molecule)

# Read the chirality of each atom
universe.atoms.chiralities

# Select atoms from their chirality
universe.select_atoms('element C and chirality S')
```

### Other enhancements

Thanks to [Atharva Kulkarni][], [`MDAnalysis.units.convert`][] knows about milliseconds and microseconds. The function, as the name indicates, converts quantities from one unit to an other.

> @edisj improved the reading of H5MD files so it can better read files without a topology. For these files, the number of atoms is now read from the positions, velocities, or forces in the trajectory.

Author: I did not think to look at the AUTHORS file :stuck_out_tongue:

Thanks to [Richard Gowers][], again, one can apply the minimum image convention to a collection of vectors using `MDAnalysis.lib.distances.minimize_vectors`.

> This version of MDAnalysis is compatible with TPR files produced with Gromacs 2022.

Reviewer: ```suggestion
Thanks to [Jonathan Barnoud][], this version of MDAnalysis is compatible with TPR files produced with Gromacs 2022.
```
Let's not forget you!

```python
import MDAnalysis as mda
from rdkit import Chem

molecule = Chem.MolFromSmiles('CC[C@H](C)[C@H](C(=O)O)N')
universe = mda.Universe(molecule)

# Read the chirality of each atom
universe.atoms.chiralities

# Select atoms from their chirality
universe.select_atoms('element C and chirality S')
```

### Other enhancements

Thanks to [Atharva Kulkarni][], [`MDAnalysis.units.convert`][] knows about milliseconds and microseconds. The function, as the name indicates, converts quantities from one unit to an other.

[Edis Jakupovic][] improved the reading of H5MD files so it can better read files without a topology. For these files, the number of atoms is now read from the positions, velocities, or forces in the trajectory.

Thanks to [Richard Gowers][], again, one can apply the minimum image convention to a collection of vectors using `MDAnalysis.lib.distances.minimize_vectors`.

Thanks to [Jonathan Barnoud][], this version of MDAnalysis is compatible with TPR files produced with Gromacs 2022.

> 

Reviewer 2: ```suggestion

Thanks to [Hugo MacDermott-Opeskin][], you can now specify additional compiler flags for the C/C++ compiler to customise your installation. This can be used to improve performance by using architecture specific flags. See the [detailed instructions in the User Guide][] for more information.

```

## Author statistics

Altogether, this represents the work of 14 authors, 6 of which were new contributors:

- @melomcr
- @mdd31
- @scal444
- @edisj
- @Atharva7K
- @yangtcai

MDAnalysis thanks [NumFOCUS][]'s continued support as the organisation's fiscal sponsor.

- The MDAnalysis Team

[installation instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html
[conda-forge channel]: https://anaconda.org/conda-forge/mdanalysis
[PyPi]: https://pypi.org/project/MDAnalysis/
[NumFOCUS]: https://www.numfocus.org
[CHANGELOG]: https://github.com/MDAnalysis/mdanalysis/blob/release-2.1.0/package/CHANGELOG
[NEP 29]: https://numpy.org/neps/nep-0029-deprecation_policy.html
[Mattia Felice Palermo]: @mattiafelice-palermo
[Philip Loche]: @PicoCentury
[`MDAnalysis.units.convert`]: https://docs.mdanalysis.org/stable/documentation_pages/units.html?highlight=unit#MDAnalysis.units.convert
[Atharva Kulkarni]: @Atharva7K
[Richard Gowers]: [@richardjgowers]
[Edis Jakupovic]: [@edisj]

> [Jonathan Barnoud]: [@jbarnoud]

Reviewer 2: ```suggestion
[Jonathan Barnoud]: [@jbarnoud]
[Hugo MacDermott-Opeskin]: [@hmacdope]
[detailed instructions in the User Guide]: https://userguide.mdanalysis.org/stable/installation.html#custom-compiler-flags-and-optimised-installations

```

---
layout: post
title: MDAnalysis 2.1 is out
---

Six months after the 2.0 release, we are happy to release the version 2.1.0 of MDAnalysis!

> With this version, we try to release new version more often. We also aim to follow [NEP 29][] that recommands what version of python and numpy to support. MDAnalysis 2.1.0 supports python 3.7 to 3.10 and requires numpy 1.18.0 at the minimum. This means our support window is still a bit larger than the recommendations, but expect that window to skink in the future as supporting older version of python and numpy has a cost both in terms of time and opportunities.

Reviewer: ```suggestion
In line with ongoing attempts to keep MDAnalysis closer to [NEP 29][], this version now only supports python 3.7 to 3.10, in addition to a minimum numpy version of 1.18.0.

Further details about MDAnalysis' support policy, including our release schedule, will be providing in an upcoming blog post.
```
